### DISCUSSION TITLE
Why not V? github.com/vlang/v

### DISCUSSION THREAD

**Discussion by X1aSheng** [carbon-language/carbon-lang#1487] [General]
why not v?
so simple, nice grammar

**Comment by sanxiyn** [comment]
I had a brief look at V. Carbon's most important requirement is C++ interoperability, and I couldn't find much about V's C++ (not C) interoperability story.

There is [c2v](https://github.com/vlang/c2v), a C/C++ to V translator, but its support for C++ is unclear. It uses Clang, so in principle it could handle C++. But, for example, it does not look at base class at all, and it is unclear how it would be translated, given that "V has no inheritance" is an advertised feature of V.
